plugins/upload/README.md

```diff
@@ -1,6 +1,7 @@
 ![plugin-upload](banner.png)
 
 Upload files from disk to a remote server over HTTP.
+Download files from a remote HTTP server to disk
```

Comment:
```suggestion
Download files from a remote HTTP server to disk.
```

A bit picky but that triggered me :D

Reply:
haha, my bad
please check the commit now